tests/examples/test_examples.py

```diff
@@ -198,7 +201,9 @@ def _assert_snapshot(example, url, example_type):
 
     assert success, "%s failed to load" % example.relpath
     assert no_resources, "%s failed with %d missing resources" % (example.relpath, len(resources))
-    assert no_errors, "%s failed with %d errors" % (example.relpath, len(errors))
+
+    if example_type != "server":
```

Comment:
ok. as a work-around, we could add a blocking time.sleep to the ``test_server_examples`` code to ensure the doc is complete. While ugly it would let us get the server examples working again.